runtime/src/lib.rs

```diff
@@ -813,6 +813,13 @@ impl pallet_block_reward::Config for Runtime {
 	type IsLegalOfficer = LoAuthorityList;
 }
 
+impl pallet_utility::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeCall = RuntimeCall;
+	type PalletsOrigin = OriginCaller;
+	type WeightInfo = pallet_utility::weights::SubstrateWeight<Runtime>;
```

Comment:
Use the custom weights instead of the provided ones.

Reply:
Thanks for pointing the error, fixed.